packages/dapp/src/utils/server.js

```diff
@@ -0,0 +1,16 @@
+const axios = require('axios').default;
+const Constants = require('./constants');
+
+const server = axios.create({
+  // @todo is there an .env var that we can use here to determine environment?
```

Comment:
I think this would depend on where we're deploying this application, if we're using Vercel, i guess it exposes a `NODE_ENV=production`

Reply:
I don't see a need for Vercel? Can't we just deploy on our own little instance (can even throw up a little EC2 machine for you)

What is the advantage to a service like Vercel in this context?

Reply:
No i meant wherever we deploy would just need to know the .env var